kore/src/main/scala/org/kframework/definition/outer.scala

```diff
@@ -368,6 +368,13 @@ case class Module(val name: String, val imports: Set[Import], localSentences: Se
     case _ =>
   }
 
+  def checkAtts (other: Set[Sentence]) : Boolean = {
+    val atts = sentences.collect({ case p: Production => p.att }).toSet
+    val otherAtts = other.collect({ case p: Production => p.att }).toSet
+    val diff = atts -- otherAtts
+    diff.isEmpty
```

Comment:
this is... not how I would recommend doing this. You are checking that the set of all attributes on one module is a subset of the set of all attributes on another module, but you are not actually checking that the same sentences have the same attributes.

Reply:
I probably should check this in this function, but given this use
`if (newSentences != m.localSentences || !m.checkAtts(newSentences))`
we already know the two sets are "equal", no? And therefore, we are safely comparing that the sets of attributes are equal for the same sentences in the two modules. 

Although, I agree that there are probably better ways to compare them.